types/views/views.go

```diff
@@ -168,6 +172,22 @@ func (v SliceView[T, V]) SliceTo(i int) SliceView[T, V] { return SliceView[T, V]
 // Slice returns v[i:j]
 func (v SliceView[T, V]) Slice(i, j int) SliceView[T, V] { return SliceView[T, V]{v.ж[i:j]} }
 
+// SliceMapKey represents a unique key for a slice, based on its address and
```

Comment:
Can we add some more emphasis on this being a thing you can use to key an _immutable_ slice?